frontend/src/components/Forms/Fields/MoneyField/index.tsx

```diff
@@ -24,7 +24,15 @@ import { IMoneyField } from "types";
  * />
  */
 
-export const MoneyField = ({ onChange, id, value, label, helperText, error }: IMoneyField) => {
+export const MoneyField = ({
+  onChange,
+  id,
+  value,
+  label,
+  helperText,
+  error,
+  required,
+}: IMoneyField) => {
   useEffect(() => {
```

Comment:
just making sure you are aware of this warning with react and useEffect

Reply:
Yes I am, thinking of a solution.